modules/objdetect/include/opencv2/objdetect/aruco_board.hpp

```diff
@@ -79,8 +79,8 @@ class CV_EXPORTS_W Board {
      * @param objPoints Vector of vectors of board marker points in the board coordinate space.
      * @param imgPoints Vector of vectors of the projections of board marker corner points.
      */
-    CV_WRAP void matchImagePoints(InputArrayOfArrays detectedCorners, InputArray detectedIds,
-                                  OutputArray objPoints, OutputArray imgPoints) const;
+    CV_WRAP virtual void matchImagePoints(InputArrayOfArrays detectedCorners, InputArray detectedIds,
+                                          OutputArray objPoints, OutputArray imgPoints) const;
```

Comment:
Board implements Pimpl pattern. Virtual is redundant here.

Reply:
Move implementation to pimple?